test/WebJobs.Extensions.CosmosDB.Tests/Trigger/CosmosDBTriggerAttributeBindingProviderTests.cs

```diff
@@ -105,7 +111,8 @@ public async Task ValidParametersWithAppSettings_Succeed(ParameterInfo parameter
             CosmosDBTriggerBinding binding = (CosmosDBTriggerBinding)await provider.TryCreateAsync(new TriggerBindingProviderContext(parameter, CancellationToken.None));
 
             Assert.Equal(typeof(IReadOnlyList<Document>), binding.TriggerValueType);
-            Assert.Equal(binding.DocumentCollectionLocation.Uri, new Uri("https://fromSettings"));
+            Assert.Equal(new Uri("https://fromSettings"), binding.DocumentCollectionLocation.Uri);
+            Assert.Equal(new Uri("https://fromSettings"), binding.LeaseCollectionLocation.Uri);
```

Comment:
I made sure that we validate the lease URI in each of these tests no. Some of these were falling back to use "https://fromEnvironment" when they shouldn't be.

Reply:
Good catch, I didn't notice it in the refactor in the previous PR